### ISSUE TITLE
Switch to maven project

### ISSUE THREAD

**Issue by AKryukov92** [LordStephen77/DreamBlocks-Beyond-of-Time#13]
Maven is excellent tool for managing dependencies and configuring project compilation.
It is supported by all modern java IDE. It is also supported by CI tools, which might be used in future.

I'm going to migrate to maven. Maybe it will require changing structure of whole project.

**Comment by AKryukov92** [comment]
@LordStephen77 Why do you use com.dnsalias.java.timer.AdvancedTimer?
Why not just to use `System.currentTimeMillis();` ?

**Comment by LordStephen77** [comment]
@AKryukov92 you are right. I wanted to use Gradle but I tested Maven and find it better and easier to handle. You do not have to reorganize your project, just create a new maven project with Eclipse and copy the project inside keeping the tree of the current folders. It is very easy. If you want me to do it. After public the project with maven.



it's true! In the old project I used `System.currentTimeMillis();`

The `java.lang.System.currentTimeMillis()` method returns the current time in milliseconds.The unit of time of the return value is a millisecond, the granularity of the value depends on the underlying operating system and may be larger.

For example, many operating systems measure time in units of tens of milliseconds.


## Declaration:

Following is the declaration for `java.lang.System.currentTimeMillis()` method

`public static long currentTimeMillis()`


## Parameters:

NA

## Return Value

This method returns the difference, measured in milliseconds, between the current time and midnight, January 1, 1970 UTC(coordinated universal time).


## Exception

NA


## Example:

The following example shows the usage of `java.lang.System.currentTimeMillis()` method.

```
package com.lgs.example;

import java.lang.*;

public class SystemDemo {

   public static void main(String[] args) {

      // returns the current time in milliseconds
      System.out.print("Current Time in milliseconds = ");
      System.out.println(System.currentTimeMillis());
   }
}

```

I probably used the small timer library found in the lib (timer.jar) folder. I want to eliminate any addiction leaving only the standard library. I've been thinking about using LWJGL3 in a small way! Is a very complete library and contains many modules. But it uses opengl and I'm not familiar with opengl programming (I just experienced a couple of times).

Include the project in maven, then you will be configured to take all the dependencies you want to include. Thanks for your help.

**Comment by LordStephen77** [comment]
@AKryukov92 I have published the integrated project in gradle. I just need to understand how to implement the dependencies I have in the lib folder. Some libraries are old, I used them because I knew them. With git and Gradle I have little experience I admit. If you want to control it, let me know that I have combined. With all the work I have in this period I crazy!

Thank you so much for your help, I appreciate it a lot.
